requirements/dev.txt

```diff
@@ -4,6 +4,10 @@
 #
 #    make upgrade
 #
+--index-url http://edx.devstack.devpi:3141/root/pypi/+simple/
```

Comment:
wtf is this? ok I can remove it manually eventually unless we want it for some reason

Reply:
That's pip saying that it's ok to search our local devpi servers for packages to install, it's configured in `/root/.pip/pip.conf` inside our containers.  However, it seems to never work as intended on travis.  I usually just strip it out manually, but I wish there was a better way to handle this.